src/app/sessions/reducers/index.ts

```diff
@@ -50,14 +50,22 @@ export const getSessionCreationEntitiesState = createSelector(
     state => state.entities
 );
 
-export const getRecentlyCreatedSessionId = createSelector(
+export const getRecentlyCreatedTransactionId = createSelector(
     getSessionCreationState,
-    fromSessionCreate.getRecent
+    fromSessionCreate.getRecent,
+);
+
+export const getRecentlyCreatedSessionId = createSelector(
```

Comment:
is this still used anywhere?

Reply:
It is in sessions-create.component. This id is used to filter the problems from state to display only the ones associated with the created session.

session created -> Get problems for session -> add problems to ProblemsState -> Get problems for session from ProblemsState